frontend/src/goutil

```diff
@@ -15,6 +16,11 @@ GOBIN=go
 if ! command -v go &> /dev/null
 then
   GOBIN=/usr/lib/go-1.13/bin/go
+else
+  # Temporary solution until https://github.com/golang/go/issues/28194 is fixed
+  # in order to retry failed fetch requests.
+  # GOPROXY fallback was added in Go 1.15
+  export GOPROXY="proxy.golang.org|proxy.golang.org|direct"
```

Comment:
should we be worried about go 1.14?
The other branch of the `if` is for go 1.13, but the functionality you're using here is for 1.15+

Reply:
changed the logic, now based on the output of `go version`